### ISSUE TITLE
[Feat] Docker Image

### ISSUE THREAD

**Issue by mahyarmirrashed** [andrewbanchich/shreddit#109]
A Docker/Podman image would be great as something to have periodically run on a Linux server.

**Comment by mahyarmirrashed** [comment]
k8s workflows

**Comment by andrewbanchich** [comment]
Another reason might be security. Instead if auditing all the code you could just run inside a container.

That being said, it could be as easy as doing a volume binding to your local binary and doesn't need a custom image.

**Comment by scottjl** [comment]
just seems like overkill. if you're that worried about the code for this app, are you running _every_ 3rd party app in a container? if you're running k8 then you should be building your own containers anyway.

**Comment by andrewbanchich** [comment]
Yeah, I agree with that. Probably not something in scope for this project.
